pkg/mountutil/mountutil.go

```diff
@@ -55,72 +55,84 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		src, dst string
 		options  []string
 	)
-
-	split := strings.Split(s, ":")
-	switch len(split) {
-	case 1:
-		dst = s
-		res.AnonymousVolume = idgen.GenerateID()
-		logrus.Debugf("creating anonymous volume %q, for %q", res.AnonymousVolume, s)
-		anonVol, err := volStore.Create(res.AnonymousVolume, []string{})
+	if runtime.GOOS == "windows" {
+		sr, ds, optns, err := ProcessSplit(s, volStore, res, src, dst, options)
```

Comment:
Why are adding a check for goos if we are doing the process split logic in os specific files?  it feels strange to have the case statements tucked away for Windows but for other OS (linux) process split is in this file especially since process split doesn't do anything for linux/freebsd and are exactly the same

Reply:
I will just keep the process split logic to respective os files and remove the os logic.